README.md

```diff
@@ -44,8 +44,6 @@ This is a JavaScript project for the Leaderboard list app, using webpack and ES6
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
```

Comment:
- [ ] @chiedozie013 as previously requested kindly remove the live demo section, since you currently don't have a live demo
![image](https://github.com/chiedozie013/Leaderboard/assets/61408860/f06246b7-b01b-4395-beb9-5569e5664807)

- [ ] [optional] It would have been nice to correct instructions on how to run the project after cloning it, since this is a webpack project users are supposed to use `npm i` to install dependencies and then run `npm start` to start the project